docs/server-versions-dependencies.md

```diff
@@ -72,13 +66,15 @@ Temporal emits metrics by default in a format that is supported by Prometheus. M
 
 ### Multi-cluster replication
 
-This is an experimental feature, most Temporal users do not need this. Requires the use of event streaming software. See the note on event streaming dependency below.
+This is an experimental feature, most Temporal users do not need this. You should be on the latest Temporal version (v1.7+) to use this. Requires the use of event streaming software. See the note on event streaming dependency below.
 
-- **Kafka v2.1.1 & Zookeeper v3.4.6**
+- **Zookeeper v3.4.6**
 
```

Comment:
kafka was removed in v1.4, which is no longer supported by us per our own versioning policy

Reply:
So we no longer support versions that supported Kafka?

Reply:
aye

Reply:
Cool - then let's remove it from above as well? (Elasticsearch section).

Reply:
no the removal was only for the multi cluster replication feature, kafka is still valid for visibility as of  v1.5.4